src/bnb_trading/data_fetcher.py

```
@@ -171,38 +173,30 @@
             Dict с daily и weekly DataFrames
         """
         try:
             # Изчисляваме timestamps
             end_time = self.exchange.milliseconds()
             start_time = end_time - (lookback_days * 24 * 60 * 60 * 1000)
-            
+
             logger.info(f"Извличане на {lookback_days} дни BNB данни...")
-            
+
             # Извличаме daily данни (Binance limit е max 1000)
```

Ruff (0.12.2)

Comment contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF003)